splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -2148,276 +2149,218 @@ public static void SHOW_CREATE_TABLE(String schemaName, String tableName, Result
     {
```

Comment:
I happened to touch system procedures too when implementing local temporary tables, so I think maybe I should to put the questions and observations I had here. None of them are really critical and you can just ignore if you think it's not worth implementing.